src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = ",";
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private int amountBuy = 0;
+    private int amountSupply = 0;
+    private int result;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        amountBuy = 0;
+        amountSupply = 0;
+        result = 0;
+
+        readFile(fromFileName);
+        calculateResult();
+        writeIntoFile(toFileName);
+    }
+
+    public void readFile(String fromFileName) {
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(DELIMITER);
+                String operationType = parts[0].trim();
+                int amount = Integer.parseInt(parts[1].trim());
+
+                if (operationType.equals(BUY)) {
+                    amountBuy += amount;
+                } else if (operationType.equals(SUPPLY)) {
+                    amountSupply += amount;
+                }
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read the file", e);
+        }
+    }
+
+    public void calculateResult() {
+        result = amountSupply - amountBuy;
+    }
```

Comment:
This method should return something that will be provided to method writeIntoFile it could be a list with results (amountBuy, amountSupply, result) or array or map, or even custom result object